apps/fem/poisson3.sim

```diff
@@ -0,0 +1,188 @@
+%
+%  Strong form (poisson problem)
+%
+% - \nabla^2 u = f(x,y) in \Omega (domain)
+%            u = 0      on \Gamma (boundary)
+%
+%  solved using linear functions on a tetrahedral mesh
+%
+
+element Node
+    x    : vector[3](float); % position
+    edge : bool;             % true if part of the boundary
+    u    : float;            % solution
+    i    : int;              % index of node
```

Comment:
Sorry... my bad. It is actually used in the "print result" function. It dumps `vtk` files, which first list nodes, then elements are connected by node indices. In order to loop over elements (tetrahedrons), and dump its 4 corner indices, each node must be aware of its own index in the global system.

That is, if there is not some function call I can make to `Set::getElementIndex(ElementRef el)`, but I could not find any such function.